docs/3-tutorials/01-run-collator.md

```diff
@@ -1,5 +1,168 @@
-# How to run a collator
+# Collator Management
+This tutorial will show how create and maintain a collator node in Horizen 2.
 
-[ coming soon... ]
+## How to Run a Collator
+The following are the steps required for running a collator node:
+- Run a Horizen 2 node (see [TBD] for details). Make sure to add the following command-line parameter: `--collator`.
+- Create an account for the collator, e.g. by using Metamask. It will be used for joining the collator pool and for receiving the block rewards. Make sure that it has a balance with at least the minimum stake required plus some extra for transaction fees. 
+- Import the account in Polkadot.js as described in [How to Import an Ethereum Account into Polkadot.js Apps](./05-add_account_polkadotjs.md) tutorial. Note: if you don't use Polkadot.js for interacting with your node, you can skip this step.
+- Create the session keys on the collator node:
+    - Open Polkadot.js Apps and connect it to the collator node.
+    - Select _Developer>RPC Calls_. 
+    - Select _author_ as endpoint and then _rotateKeys_. 
+    - Press _Submit RPC call_.
+    - Copy the string representing the public session key.
+- Map the collator account to the session keys created in the previous step:
+    - On Polkadot.js Apps, select _Developer>Extrinsics_.
+    - In _using the selected account_ field, select the collator account.
+    - Select _session_ and then _setKeys_.
+    - Put the session key saved before in _keys_ field and put _0x_ in _proof_ field.
```

Comment:
I've checked the polkadot-sdk and it's indeed completely ignored. I feel fooled :D